bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala

```diff
@@ -494,8 +483,8 @@ object TpeGhostIfuFeature {
 
 
   // Color for IFU limts.
-  private val IfuFovColor: Color = Color.RED
-  private val PatrolRangeColor: Color = OtColor.SALMON
+  val IfuFovColor: Color = Color.RED
```

Comment:
I think it was requested this way but the red labels make me think an error is being indicated.